pkg/trace/config/apply.go

```diff
@@ -278,10 +279,14 @@ func (c *AgentConfig) applyDatadogConfig() error {
 		c.ReceiverHost = "0.0.0.0"
 	}
 
+	var grpcPort int
+	if otlp.IsEnabled(config.Datadog) {
+		grpcPort = config.Datadog.GetInt("experimental.otlp.internal_traces_port")
+	}
+
 	c.OTLPReceiver = &OTLP{
 		BindHost:        c.ReceiverHost,
-		HTTPPort:        config.Datadog.GetInt("experimental.otlp.http_port"),
```

Comment:
If you removed this, you should remove everything else related to it too, such as the HTTP server in `./pkg/trace/api/otlp.go`

Reply:
Do you think it is preferable to use gRPC or HTTP for the core Agent to trace Agent communication? And, is it possible that we could change our mind about it?

Reply:
Good question! I have no idea. For some reason I'd suspect gRPC might be better managed for it and might even use HTTP/2. It's hard to know without any actual testing. I guess it's fine to leave the setting and implementation there in case we want to use it later!

I'll resolve this comment, no change needeed.